### PR TITLE
[Issue 2] Fixed graph scale and added title/axis labels

### PR DESCRIPTION
**Pull request by MrGribel**
Small changes to the update function to display some missing information on the graph during plotting.
The team could decide on different styles of visualization in the future.
graph

### CODE REVIEW

File: ButtonFunctions/update.py [hunk 1]
```diff
@@ -17,13 +17,16 @@ def update(application):
...
+        time = float(time)
         application.forces.append(force)
         application.times.append(time)
         ydata = np.asarray(application.forces)
         xdata = np.asarray(application.times)
         application.plot_pannels['plot2'].plot(xdata, ydata, clear=True, _callSync='off')
-
+        application.plot_pannels['plot2'].setTitle("Thrust Curve")
+        application.plot_pannels['plot2'].setLabel('left', 'Force (N)')
+        application.plot_pannels['plot2'].setLabel('bottom', 'Time (s)')
```
Comment (PatrickSampaioUSP) [inline ButtonFunctions/update.py]: I believe that this modification would be more appropriate in the Application class, cause is a configuration issue. The update class should concern about data issues.

**Comment by MrGribel** [comment]
Patrick, I've moved the modifications you've referenced to a new method inside the Applications class, called plot_information.
I believed it was more appropriate to introduce a new one so that the pannels_initialization method don't get too cluttered with labels/titles once we start adding new plots.

**Comment by PatrickSampaioUSP** [comment]
I liked the modifications to the Application class, it's easy to configure other plots. I will test on my machine today and if everything is ok I will merge this PR to the master.

**Comment by PatrickSampaioUSP** [comment]
However, I would like to you take a look on this (https://docs.python.org/3/library/typing.html), it's a tool for documenting the code.

**Pull request merged**